lib/streamlit/web/cli.py

```diff
@@ -252,7 +252,6 @@ def cache():
 @cache.command("clear")
 def cache_clear():
     """Clear st.cache, st.memo, and st.singleton caches."""
-    import streamlit.legacy_caching
     import streamlit.caching
```

Comment:
nit: could the import of `streamlit.caching` be removed here too? or are they subtly different somehow?

Reply:
Yes, good catch!